osu.Game/Screens/Ranking/Statistics/PerformanceBreakdownChart.cs

```diff
@@ -0,0 +1,243 @@
+                                        Font = OsuFont.GetFont(weight: FontWeight.Regular, size: 18),
+                                        Text = "Achieved PP",
+                                        Colour = Color4Extensions.FromHex("#66FFCC")
+                                    },
+                                    achievedPerformance = new OsuSpriteText
+                                    {
+                                        Origin = Anchor.CentreRight,
+                                        Anchor = Anchor.CentreRight,
+                                        Font = OsuFont.GetFont(weight: FontWeight.SemiBold, size: 18),
+                                        Colour = Color4Extensions.FromHex("#66FFCC")
+                                    }
+                                },
+                                new Drawable[]
+                                {
+                                    new OsuSpriteText
+                                    {
+                                        Origin = Anchor.CentreLeft,
+                                        Anchor = Anchor.CentreLeft,
+                                        Font = OsuFont.GetFont(weight: FontWeight.Regular, size: 18),
+                                        Text = "Maximum",
+                                        Colour = OsuColour.Gray(0.7f)
+                                    },
+                                    maximumPerformance = new OsuSpriteText
+                                    {
+                                        Origin = Anchor.CentreLeft,
+                                        Anchor = Anchor.CentreLeft,
+                                        Font = OsuFont.GetFont(weight: FontWeight.Regular, size: 18),
+                                        Colour = OsuColour.Gray(0.7f)
+                                    }
+                                }
+                            }
+                        },
+                        chart = new GridContainer
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            Origin = Anchor.TopCentre,
+                            Anchor = Anchor.TopCentre,
+                            ColumnDimensions = new[]
+                            {
+                                new Dimension(GridSizeMode.AutoSize),
+                                new Dimension(),
+                                new Dimension(GridSizeMode.AutoSize)
+                            }
+                        }
+                    }
+                }
+            };
+
+            spinner.Show();
+
+            new PerformanceBreakdownCalculator(playableBeatmap, difficultyCache, performanceCache)
+                .CalculateAsync(score, cancellationTokenSource.Token)
+                .ContinueWith(t => Schedule(() => setPerformanceValue(t.GetResultSafely())));
```

Comment:
Feels like this should maybe be moved to `LoadComplete` to avoid potentially calculating before the user requests details statistics. But I guess it's fine as-is until we notice an issue.

Reply:
In my testing, it only starts calculation when the detailed statistics panel is first shown, so it should be fine.